README.md

```diff
@@ -67,6 +67,9 @@ To test components locally in the context in which they will be used we use `npm
 3. Run npm link
 4. Run npm link path/to/your/consumer-context/node_modules/react (example npm link ../consumer-context/node_modules/react)
 5. Navigate to the consumer context folder.
-6. Run npm link data-transparency-ui
+6. Update nvm version if needed (example nvm use 12)
```

Comment:
Can we remove these updates regarding the node versions? I created https://github.com/fedspendingtransparency/data-transparency-ui/pull/103 to bump the node version to 14.

Maybe we can add the node version we are on in the readme?